### DISCUSSION TITLE
Issue with Velero Restore

### DISCUSSION THREAD

**Discussion by bkrrajmali** [vmware-tanzu/velero#8073] [Community support Q&A]
I am currently working on a Proof of Concept (POC) involving Velero for backing up and restoring OpenShift namespaces. During this process, I encountered an issue when attempting to restore a deleted namespace. The details of the issue are as follows:

Issue Description:

Phase: Failed
Error Message: Phase: Failed (run 'velero restore logs backup01-20240801073158' for more information) An error occurred: file not found
Steps Taken:

Deleted the OpenShift namespace.
Attempted to restore the namespace using Velero.
Despite following the standard procedure, the restore process fails with the error mentioned above.

Logs:
I have run the suggested command 'velero restore logs backup01-20240801073158' and attached the output log file for your reference.

Could you please assist in diagnosing and resolving this issue? Any guidance or suggestions you can provide will be greatly appreciated.

Thanks
Bala
[attach.log](https://github.com/user-attachments/files/16450833/attach.log)

**Comment by sseago** [comment]
Since the restore failed, it probably didn't get far enough in the process to upload logs for the restore. A "Failed" restore generally means there was a fatal error early in the process, as opposed to a "PartiallyFailed" restore that encountered some errors along the way but still managed to run to completion. I would check velero pod logs in this case to look for the source of the error.

**Comment by bkrrajmali** [comment]
[velero-85b498bf44-hwzks-velero.log](https://github.com/user-attachments/files/16505597/velero-85b498bf44-hwzks-velero.log)

Uploaded the log file

**Comment by bkrrajmali** [comment]
Hi
I have uploaded the logs.
Thanks
Bala

**Reply by sseago** [reply]
There are no log messages related to backing up or restoring in that log, so it looks like the pod didn't process any during the time this pod has been up. Also, the log messages only span about four minutes, which means Velero has only recently started. If things are consistently failing for you like this, you would need to capture the pod logs while the pod is still running. Is there any reason that velero had only started 4 minutes before you grabbed the logs? was it shut down manually, or is it crashing consistently?
